README.md

```diff
@@ -334,9 +334,52 @@ Please see the [RAPIDS Release Selector](https://rapids.ai/start.html) for more
 ## Benchmarking
 cuSignal uses pytest-benchmark to compare performance between CPU and GPU signal processing implementations. To run cuSignal's benchmark suite, **navigate to the topmost python directory ($CUSIGNAL_HOME/python)** and run:
 
-`pytest --benchmark-only`
-
-As with the standard pytest tool, the user can use the `-v` and `-k` flags for verbose mode and to select a specifc benchmark to run. When intrepreting the output, we recommend comparing the _minimum_ execution time reported.
+`pytest --benchmark-enable --benchmark-gpu-disable`
+
+Benchmark are disabled by default in `setup.cfg` providing only test correctness checks.
+
+As with the standard pytest tool, the user can use the `-v` and `-k` flags for verbose mode and to select a specifc benchmark to run. When intrepreting the output, we recommend comparing the _mean_ execution time reported.
+
+To minimize CLI cluster from benchmark results add `--benchmark-columns=LABELS`
+
+Parameter `--benchmark-gpu-disable` is to disable memory checks from Rapids GPU benchmark tool. https://github.com/rapidsai/benchmark
+Doing so speeds up benchmarking.
+
+If you wish to skip benchmarks of SciPy functions add `-m "not cpu"`
+
+Lastly, benchmarks will be executed on local files. Therefore to test recent changes made to source, rebuild cuSignal.
+
+### Example
+`pytest -k _upfirdn2d -m "not cpu" --benchmark-enable --benchmark-gpu-disable --benchmark-columns=mean`
```

Comment:
The only confusing part to me is use of `_upfirdn2d` here rather than `upfirdn2d`. We know this is because `_upfirdn2d` is a custom kernel, but let's make this more clear to the users? Would it be better to give an example of `upfirdn2d`?

Reply:
Fixed